sct.py

```diff
@@ -45,9 +47,19 @@ def install_callback(ctx, _, value):
     return sys.exit(0)
 
 
+def install_package_from_dir(ctx, _, directories):
```

Comment:
Why is there `_` in the statement of function?

Reply:
cause there are more parameters pass in this callback, and we are not using them.